src/firebase-db.ts

```diff
@@ -228,32 +228,38 @@ export function createOrUpdateAnswer(answer: IExportableAnswerMetadata) {
 
   let answerDocData: LTIRuntimeAnswerMetadata | AnonymousRuntimeAnswerMetadata;
 
+  const commonFields = {
```

Comment:
I think you should type this to get some extra compile type protection.

Reply:
I just reverted this change so we can get type checking. I didn't try it but I figured any typing of this would add more lines of code than just duplicating the 3 fields twice.